Sources/FluidAudioCLI/Commands/AsrBenchmark.swift

```diff
@@ -121,6 +121,9 @@ public class ASRBenchmark {
                 if config.debugMode && index > 0 {
                     logger.info("   🔍 Processing file \(index + 1)")
                 }
+                
+                // Reset decoder state for each new file
+                try await asrManager.resetDecoderState(for: .microphone)
```

Comment:
@Alex-Wengg fyi, we were resetting the state too aggressively. Its fine for benchmarking but in a real world use case it breaks down